ViewModelTestingDemo/Podfile

```diff
@@ -0,0 +1,22 @@
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '10.0'
+use_frameworks!
+
+target 'ViewModelTestingDemo' do
+    pod 'AlamofireImage', '~> 3.1'
+    pod 'Intrepid', '~> 0.6.1'
+end
+
+target 'ViewModelTestingDemoTests' do
+    pod 'AlamofireImage', '~> 3.1'
+    pod 'OHHTTPStubs'
+    pod 'OHHTTPStubs/Swift'
+end
```

Comment:
Make this a subtarget and inherit the search paths so the AlamofireImage isn't duplicated